src/Serialization/Fields.jl

```diff
@@ -276,6 +276,81 @@ function load_internal_with_parent(s::DeserializerState,
     return parent(num, den)
 end
 
+################################################################################
+# RationalFunctionField
+
+encodeType(::Type{<:AbstractAlgebra.Generic.RationalFunctionField}) = "RationalFunctionField"
+reverseTypeMap["RationalFunctionField"] = AbstractAlgebra.Generic.RationalFunctionField
```

Comment:
Since you merged my serialization PR, ideally these two lines and similar ones in this PR should also be adjusted accordingly :-)

Reply:
yup, will do that now